demo-apps/cf-android/app/src/main/java/org/eclipse/californium/examples/MainActivity.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2015 Institute for Pervasive Computing, ETH Zurich and others.
+ * Copyright (c) 2018 Institute for Pervasive Computing, ETH Zurich and others.
```

Comment:
You may add 2018, but not replace the 2015.

Reply:
yeah I will correct it